stackit/internal/resources/mongodb-flex/instance/resource_test.go

```diff
@@ -24,7 +24,6 @@ func TestAcc_MongoDBFlexInstance(t *testing.T) {
 	}
 
 	name1 := "odjtest-" + acctest.RandStringFromCharSet(7, acctest.CharSetAlpha)
-	name2 := "odjtest2-" + acctest.RandStringFromCharSet(7, acctest.CharSetAlpha)
```

Comment:
why is the renaming option removed?

Reply:
renaming instance do not work. It returns 500. If it should work then we can report a bug :)

Reply:
please report it :)